src/components/buttons.js

```diff
@@ -54,6 +57,7 @@ function ButtonBase({
 
   return (
     <button
+      // @ts-ignore
```

Comment:
This error didn't make sense to me. It looks like after adding in the button props, ts thinks this is a RefObject<HTMLInputElement>|RefObject<HTMLButtonElement>  rather than RefObject<HTMLButtonElement> and so ts complains that the type is not the same.


```
^[[A^[[Asrc/components/buttons.js:60:7 - error TS2322: Type 'RefObject<HTMLButtonElement> | RefCallback<HTMLButtonElement> | RefObject<HTMLInputElement> | RefCallback<...> | undefined' is not assignable to type 'RefObject<HTMLButtonElement> | RefCallback<HTMLButtonElement> | undefined'.
  Type 'RefObject<HTMLInputElement>' is not assignable to type 'RefObject<HTMLButtonElement> | RefCallback<HTMLButtonElement> | undefined'.
    Type 'RefObject<HTMLInputElement>' is not assignable to type 'RefObject<HTMLButtonElement>'.
      Type 'HTMLInputElement' is not assignable to type 'HTMLButtonElement'.
        Types of property 'labels' are incompatible.
          Type 'NodeListOf<HTMLLabelElement> | null' is not assignable to type 'NodeListOf<HTMLLabelElement>'.
            Type 'null' is not assignable to type 'NodeListOf<HTMLLabelElement>'.

60       ref={buttonRef}
```